testctrl/proto/test_sessions.proto

```diff
@@ -0,0 +1,185 @@
+// Copyright 2020 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package grpc.testing.benchmarking;
```

Comment:
I think "grpc.testing.benchmarking" is still too general. What if we want to add more benchmarking-related stuff in the future and grpc.testing.benchmarking is already taken?
Also, it is customary to append "v1" "v2" etc suffix to allow updating major service version.
https://github.com/grpc/grpc/blob/abe0547d3cfaba18007a1bb778dc9c0baa3f468b/src/proto/grpc/health/v1/health.proto#L20

how about `grpc.testing.benchmarking.benchmarkmanager.v1`?   (replace benchmarkmanager with however you want to call this service you're building).